app/models/user.rb

```diff
@@ -24,6 +24,7 @@ class User < ApplicationRecord
   has_many :collections, dependent: :destroy
   has_many :comments, dependent: :destroy
   has_many :email_messages, class_name: "Ahoy::Message", dependent: :destroy
+  has_many :email_authorizations, dependent: :delete_all
```

Comment:
Why not `:destroy` ?

You can check also `spec/services/users/delete_spec.rb` to make sure that you're either deleting the association or keeping it

Reply:
I went with `delete_all` because it's faster and there are no callbacks, but maybe `destroy` is better.

Also good point about the delete user service!